src/Microsoft.ML.Ensemble/OutputCombiners/BaseMultiCombiner.cs

```diff
@@ -15,7 +15,7 @@ public abstract class BaseMultiCombiner : IMultiClassOutputCombiner
     {
         protected readonly IHost Host;
 
-        public abstract class ArgumentsBase
+        public class ArgumentsBase
```

Comment:
>public class ArgumentsBase [](start = 8, length = 26)

Classes should be either abstract or sealed.

Reply:
sure.

---
In reply to: [199927554](https://github.com/dotnet/machinelearning/pull/488#discussion_r199927554) [](ancestors = 199927554)